packages/react-native-bpk-component-banner-alert/src/__snapshots__/BpkBannerAlert-test.android.js.snap

```diff
@@ -317,7 +317,7 @@ exports[`Android BpkBannerAlert dark mode should render correctly with dismissab
             accessibilityRole="button"
             accessibilityStates={Array []}
             accessible={true}
-            clickable={true}
+            focusable={true}
```

Comment:
Is this change intentional?

Reply:
Yes, this is a change in how RN's `TouchableNativeFeedback` is rendered. It doesn't affect the actual usage of the component.